src/index.js

```diff
@@ -1,5 +1,37 @@
 import $ from 'jquery'
 
 $(document).ready(() => {
-  // have fun!
-})
+  function activateTop() {
+    $('.most-word').text(``);
+    fetch('https://wordwatch-api.herokuapp.com/api/v1/top_word')
+      .then(res => res.json())
+      .then(response => {
+        $('.most-word').text(`${Object.keys(response.word)}: ${Object.values(response.word)}`);
+    });
+  }
+
+  const buddon    = document.querySelector(".the-buddon");
+  const postThis = document.querySelector(".post-this");
+
+  buddon.addEventListener("click", postThemWords);
+
+  function postThemWords() {
+    $('.word-count').append("<br>")
+    let wordArray = postThis.value.split(" ");
+    for (let word of wordArray) {
+      let data   = {word: {value: word}};
+      const pack = { method: 'POST',
+                     headers: { 'Content-Type': 'application/json' },
+                     body: JSON.stringify(data)
+                   };
+      fetch('https://wordwatch-api.herokuapp.com/api/v1/words', pack)
```

Comment:
I'd probably like to see the pack passed to a helper here, that takes care of the fetch. This function is a bit long!